resources/js/Components/Table/CharacterTable/CharacterTable.tsx

```diff
@@ -28,6 +29,7 @@ const CharacterTable = ({
     setEditId,
     setEditData,
 }: CharTablePropType) => {
+    data = orderBy(data, ['updated_at'], ['desc'])
```

Comment:
@m-triassi is it possible for BE to do the sorting on table rows initially?

Reply:
Sure ill add that now!

Reply:
@pascwhale i've added a sort query parameter to the character index, now if you pass `sort_by` in the query string you can supply one of the following fields to sort by it:  
"name,race,class,level,faction,downtime,updated_at"
you can also supply `sort_dir` as either "asc,desc", if `sort_dir` is not supplied it defaults to asc

so for example `/character?sort_by=name&sort_dir=desc` will sort by name descending. Or using ziggy's `route` function: `route('character.index', { "sort_by": "name", "sort_dir": "desc" })`

Reply:
this will be harder and should be worked on as a feature, for the scope of this task let's just do default sorting field for now

Reply:
I also added the default updated_at sorting with that feature.

Reply:
Great, thank you for doing that and adding the matching tests! I removed frontend sorting @willyyhuang.